quantum/rgb_matrix_animations/typing_heatmap_anim.h

```diff
@@ -2,72 +2,68 @@
 RGB_MATRIX_EFFECT(TYPING_HEATMAP)
 #ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
 
-void process_rgb_matrix_typing_heatmap(keyrecord_t *record) {
-    uint8_t row = record->event.key.row;
-    uint8_t col = record->event.key.col;
-    uint8_t m_row = row - 1;
-    uint8_t p_row = row + 1;
-    uint8_t m_col = col - 1;
-    uint8_t p_col = col + 1;
+// Array not large enough for PROGMEM to save firmware space on AVR
+const uint8_t k_heatmap_vals[9] = {
```

Comment:
What does this array do?  What's it point?

Reply:
internal array used to increase the heatmap value at the site of the keypress and the 9 surrounding keys.

Reply:
Ah, so, in theory, one could adjust this, if they were so inclined? 

If so, would it make sense to make this "weak" for customizability?

Reply:
correct, this could be a point of customization. Though not sure if just flagging it as weak would work though in this case as it's declared in the .h file where it is also being used too. So there might be some issues.

Reply:
Okay, for now, should be fine.  
Though, it's one thing that has been bothering me for a bit, and I'll open an issue for it, and that's customization of the animations.